src/benchmark.rs

```diff
@@ -20,7 +20,7 @@ macro_rules! u32_benchmark {
 
             b.iter(|| {
                 for key in &test_data {
-                    map.insert(key.as_slice(), 7u32);
+                    map.insert((key as &AsRef<[u32]>).as_ref(), 7u32);
```

Comment:
You're welcome :-)

In fact, I got [an ICE](https://github.com/rust-lang/rust/issues/23046#issuecomment-87323293) trying to convert the `Vec<u32>` to `[u32]` with `.as_ref()`, and I had to add the type annotation to avoid it. The indexing syntax does indeed the same thing and is shorter however, so I'll use it instead.